packages/parse5-utils/src/index.js

```diff
@@ -232,24 +226,24 @@ function findNodes(nodes, test) {
 /**
  * Looks for a child element which passes the given test
  * @param {Node[] | Node} nodes
- * @param {(node: DefaultTreeElement) => boolean} test
- * @returns {DefaultTreeElement | null}
+ * @param {(node: Element) => boolean} test
+ * @returns {Element | null}
  */
 function findElement(nodes, test) {
-  return /** @type {DefaultTreeElement | null} */ (
-    findNode(nodes, n => adapter.isElementNode(n) && test(/** @type {DefaultTreeElement} */ (n)))
+  return /** @type {Element | null} */ (
+    findNode(nodes, n => adapter.isElementNode(n) && test(/** @type {Element} */ (n)))
```

Comment:
the type annotation in `test(...)` might be inferred now since i think `isElementNode` is a type guard

Reply:
Done.